src/theme/components/card.ts

```diff
@@ -119,8 +129,68 @@ const ReservoirCard = defineMultiStyleConfig({
           bodyMargin = "0";
         }
       }
+      // These sizes are only for the "row" layout.
+      // const imageSize = size ? imageSizes[size] : {};
       return {
         base: {
+          containerType: "inline-size",
+          "@container (min-width: 0px)": {
```

Comment:
This is intentional, correct? Using media queries, you wouldn't need this clause, but it seems setting this allows it to allow shrink past the default size.

Reply:
Yes, this is to set the base styles of the elements within the container. This was done using array syntax for the media query implementation.

Reply:
Would it be possible to set up a reusable method for formatting the code block for the container styles? Something like this (please excuse any syntax errors)?
```
const setContainerStyles = (props) => {
  const { breakpoint, styles } = props;
  const minWidth = {
    base: "0px",
    mobile: breakpoints.mobile,
  };
  return `"@container (min-width: ${minWidth[breakpoint]})": { ${styles} }`;
};
```

Reply:
added this and updated styles accordingly @bigfishdesign13